java/core/src/main/java/sleeper/core/table/TableStatusProvider.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Optional;
 
 /**
- * This class can be used to cache table identities to avoid looking them up repeatedly in the table index database.
+ * This class can be used to cache tables to avoid looking them up repeatedly in the table index database.
```

Comment:
I think this could be clearer - it could be read as meaning the table data is being cached.